content/reference/authorization.md

```diff
@@ -155,3 +157,7 @@ at once.
 When the token used to authorize the request is invalid, expired or revoke, the API will return a
 401 response. When the token is valid, but doesn't have the scopes necessary to perform requested
 action, the API will return a 403 response.
+
+## Force Which Account To Authorize
```

Comment:
Maybe call it `Force authorization on certain account`
Or `Choose which account to authorize` `Prepick account for authorization`